4001-5000/LIT4960Taabbeyo.xml

```diff
@@ -1,14 +1,19 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT4960Taabbeyo" xml:lang="en" type="work">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">ተዐብዮ፡ ነፍስየ፡ ለእግዚአብሔር፡</title>
-                <title xml:lang="gez" corresp="#t1" type="normalized">Taʿabbǝyo nafsǝya la-ʾƎgziʾabǝḥer</title>
-                <title xml:lang="en" corresp="#t1">My soul magnifies God...</title>
+
+                <title xml:lang="gez" xml:id="t1">ሰአሊ፡ ለነ፡ ማርያም፡ ሰአሊ፡ ለነ፡ ማርያም፡ ኀበ፡ ወልድኪ፡ ኄር፡ መድኀኔ፡ ዓለም፡</title>
+
+                <title xml:lang="gez" corresp="#t1" type="normalized">Saʾali lana Māryām saʾali lana Māryām ḫaba waldǝki ḫer madhane ʿālam</title>
+                <title xml:lang="en" corresp="#t1">Pray for us, Mary, Pray for us, Mary, to your Good Son Saviour of the World...</title>
+
```

Comment:
I think it could be useful to keep ታዐብዮ፡ ነፍስየ፡ etc. as t2. Then it would not be printed in the article, but maybe people searching only for this expression would find this record easier.

Reply:
OK, done. Actually in the HMML catalogue it seems to have been registered as "Sa'ali lana...". "Ta'abeyyo" is the initial words of the psalm required bebore the prayer.